README.md

```diff
@@ -1454,12 +1454,12 @@ See more in the package’s [README on GitHub](https://github.com/anza-xyz/kit/t
 
 You can see all development of this library and associated GraphQL tooling in the Kit repository on GitHub.
 
--   https://github.com/anza-xyz/kit
+- https://github.com/anza-xyz/kit
 
 You can follow along with program client generator development in the `@solana-program` org and the `@codama-idl/codama` repository.
 
--   https://github.com/solana-program/
--   https://github.com/codama-idl/codama
+- https://github.com/solana-program/
+- https://github.com/codama-idl/codama
```

Comment:
Your changes here look better IMO, but just want to flag in case we end up constantly changing the spacing in this file. Not sure if there's something we can use to enforce the style here?

Reply:
Yeah I noticed some changes in other READMEs did not cause this linting change so I think this lint change is probably the most recent one and it's just not been applied everywhere yet? I could be wrong.